packages/main/src/ComboBox.ts

```diff
@@ -402,7 +402,7 @@ class ComboBox extends UI5Element implements IFormInputElement {
 	 * **Note:** If not specified, a default text (in the respective language) will be displayed.
 	 *
 	 * **Note:** The `valueStateMessage` would be displayed,
-	 * when the `ui5-combobox` is in `Information`, `Warning` or `Error` value state.
+	 * when the `ui5-combobox` is in `Information`, `Critical` or `Negative` value state.
```

Comment:
Could you search in the whole project for

`Information`, `Warning` or `Error`


it's in 6-7 places that I have missed initially

Reply:
Done.